incubator/busbar/values.yaml

```diff
@@ -0,0 +1,46 @@
+## Busbar default Values
+##
+
+## Busbar Base Images Images
+busbarDockerImage: busbario/busbar
+busbarDockerTag: latest
```

Comment:
The image and tag has a standard format we would like to see used.
```yaml
image:
  name: myapp
  tag: 1.2.3
  pullPolicy: IfNotPresent
```
You can read more about it in the review guide... https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#configuration

Reply:
Thanks for the advise mate! =)
Changing it now! =P